storage/errors.go

```diff
@@ -0,0 +1,15 @@
+package storage
+
+import "errors"
+
+var (
+	// ErrNotExists may be thrown by docs. E.g., save an unknown document.
+	//
+	// Since: 2.3
+	ErrNotExists = errors.New("document does not exist")
+
+	// ErrAlreadyExists may be thrown by docs. E.g., save a document twice.
+	//
+	// Since: 2.3
+	ErrAlreadyExists = errors.New("document already exists")
```

Comment:
Can you please order these alphabetically? It's part of our code style.

Reply:
Done. Can you point me to a document, I don't find anything about that. If this document doesn't exist, I will try to compile these rules in the Contributing in another MR.

Reply:
https://github.com/fyne-io/fyne/pull/3325